### DISCUSSION TITLE
Consider having only one executable pism

### DISCUSSION THREAD

**Discussion by aaschwanden** [pism/pism#551] [Ideas]
We used to have `pismr`, `pisms`, and `pismv` but we retired `pisms` years in lieu of `pimsr -dEIS`. How about we unify `pismr` and `pimsv` and name it just `pism`? `pismv` functionality could be implemented similarly to `pisms`.

**Comment by ckhroulev** [comment]
I already combined `pismr` and `pismv` and removed `pismv` in the `dev` branch. (Run `pismr -test G ...` to run the verification test G, etc.)

I did not (yet) rename `pismr` to `pism` to avoid breaking everyone's scripts, but I do think we should do this.

**Comment by bueler** [comment]
Yes!

**Comment by ckhroulev** [comment]
Done in version 2.2.0.
